data/tooltips.txt

```diff
@@ -157,6 +157,18 @@ tip "disruption resistance heat:"
 
 tip "drag:"
 	`Limits the maximum speed of a ship.`
+	
+tip "drive mass reference:"
+	`The mass of your ship when this drive will function as advertised.`
```

Comment:
```suggestion
	`If your ship has this mass, this drive will function as advertised.`
```
Could be changed another way, but the previous way seemed to indicate that the mass was being changed by the drive in some way.